src/app/services/clients/clients.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { ClientCompanies } from 'src/app/models/Client';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ClientsService {
+  constructor(private http: HttpClient) {}
+  fetchClientCompanies() {
```

Comment:
It'd be nice to add the return type of the method `fetchClientCompanies`.

Reply:
@Victoradukwu, I have worked on it.